homework/Vasili_Zasinets/hw0713/LookupArrayServiceTest.java

```diff
@@ -0,0 +1,52 @@
+package com.tutrit.java.quickstart.hw0713;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class LookupArrayServiceTest {
+
+    LookupArrayService lookupArrayService = new LookupArrayService();
+    int[] array;
+    int max;
+    int min;
+    int indexMin;
+    int indexMax;
+    int i;
+
+    @Before
+    public void setUp() {
+        this.lookupArrayService = lookupArrayService;
+        this.array = new int[]{1, -2, 8, 9, 6, 3};
+        this.max = 9;
+        this.min = -2;
+        this.indexMin = 1;
+        this.indexMax = 3;
+        this.i = 5;
```

Comment:
this declarations are redundant. Expected simple values could be hardcoded right in the test, it's ok. Otherwise, fabric method could be used.